sampleSpecs/test.smithy

```diff
@@ -194,3 +194,63 @@ structure ErrorDetails {
     @required
     location: String
 }
+
+apply QueryPrecedence @httpRequestTests([
+    {
+        id: "QueryPrecedence",
+        documentation: "Prefer named query parameters when serializing",
+        protocol: simpleRestJson,
+        method: "POST",
+        uri: "/Precedence",
+        body: "",
+        queryParams: [
+            "bar=named",
```

Comment:
I think this should be `bar=fromMap` (atleast one test fails because of that)

or I don't understand the nature of the test?

Reply:
@daddykotex foo is a field in a  structure annotated with httpQuery trait mapping it to the key bar

Reply:
ahhh ok, now I understand